README.md

```diff
@@ -1,2 +1,29 @@
-# index-mermaid-graph
-a tool that produces SVG diagrams using OLM graphs as the input
+## Output Mermaid graph script from Operator Lifecycle Manager indices
+
+### Required:
+make, sqlite3, go v14+, container execution environment (such as Docker, podman)
+
+### Usage
+Adjust the sqlite file pointed to in `sqlite3.sql`, this controls what index gets graphed
```

Comment:
What do you mean with that? Could you please describe the steps to do it?

Reply:
Ah, good catch, actually pointed out the README was out dated. Fixed to add mention INDEX_DB_PATH_AND_NAME variable of in 0ee2acc1